projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -258,28 +258,14 @@ function wpcom_mark_launchpad_task_complete( $task_id ) {
  * @param array $task_definitions The tasks to initialize.
  */
 function wpcom_launchpad_init_listeners( $task_definitions ) {
-	require_once WP_CONTENT_DIR . '/lib/log2logstash/log2logstash.php';
```

Comment:
If you still wanted to do this on simple wpcom for now, could you use a file_exists check before the require and function_exists check in the catch?

Reply:
Is the `IS_WPCOM` global defined on Atomic sites? That might be a simpler check... looks like that's what we do in `launchpad.php`, though that's also filtering by blog ID.

Reply:
Ah yes, `if ( defined( 'IS_WPCOM' ) && IS_WPCOM )` would work for checking simple wpcom sites. That constant shouldn't be on Atomic sites, it's how the status package checks for example:

https://github.com/Automattic/jetpack/blob/b5e05139090411f835804a3a2da0c18be9b9e57a/projects/packages/status/src/class-host.php#L61-L68